.github/workflows/brew.yml

```diff
@@ -0,0 +1,21 @@
+name: brew
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  homebrew:
+    name: Bump homebrew-core formula
+    runs-on: ubuntu-latest
+    if: ${{ needs.release-please.outputs.release_created }}
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@v2
```

Comment:
I think we would need a security review for 3rd party github actions

Reply:
Sounds good!